docs/api.md

```diff
@@ -413,6 +414,30 @@ console.log(
 
 * See [`pino-multi-stream` ⇗](https://github.com/pinojs/pino-multi-stream)
 
+<a id="level-streams"></a>
+#### `Destination: pino-multi-stream`
+
+Default destination is `STDOUT`. If you want to use different streams based on log level you can use `pino-multi-stream`.
+
+In this example we use `STDERR` for `error` and `fatal` levels and `stdout` as default for others.
+
+```js
+const pino = require('pino')
+
+const { multistream } = require('pino-multi-stream')
+var streams = [
+  {stream: process.stdout},
+  {level: 'error', stream: process.stderr},
+  {level: 'fatal', stream: process.stderr}
+]
```

Comment:
Can you make the default example using `pino.destination()` instead?

Reply:
@mcollina I'm quite new to Pino (started today) so feel free to edit my PR :)